### PR TITLE
chore: use appearance for get as well if available

### PR DESCRIPTION
**Pull request by KazuCocoa**
I cannot reproduce https://github.com/appium/appium/issues/19486 on my env with multiple combinations, but this change is what we could do as our side.

### CODE REVIEW

File: WebDriverAgentLib/Categories/XCUIDevice+FBHelpers.m [hunk 1]
```diff
@@ -356,6 +356,14 @@ - (BOOL)fb_setAppearance:(FBUIInterfaceAppearance)appearance error:(NSError **)e
 
 - (NSNumber *)fb_getAppearance
 {
+#if __clang_major__ >= 15 || (__clang_major__ >= 14 && __clang_minor__ >= 0 && __clang_patchlevel__ >= 3)
```
Comment (mykola-mokhnach) [inline WebDriverAgentLib/Categories/XCUIDevice+FBHelpers.m:359]: should __clang_minor__ be >= 3

Comment (KazuCocoa) [reply WebDriverAgentLib/Categories/XCUIDevice+FBHelpers.m:359]:
```
% ./clang --version
Apple clang version 14.0.3 (clang-1403.0.22.14.1)
Target: arm64-apple-darwin22.6.0
Thread model: posix
InstalledDir: /Applications/Xcode_14.3.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/.
```

Comment (mykola-mokhnach) [reply WebDriverAgentLib/Categories/XCUIDevice+FBHelpers.m:359]: ok, I thought it was a typo

**Review by mykola-mokhnach — APPROVED** [review]

**Comment by KazuCocoa** [comment]
let me merge this as this change itself won't bring new issues

**Pull request merged**